### ISSUE TITLE
Hard to find Information about this Module

### ISSUE THREAD

**Issue by PythonLinks** [MatthieuDartiailh/bytecode#150]
First thank you hugely for writing and maintaining this. 

I would like to read more about this module.  See what people have done with it, whether they like it or not.  So I did a search on google for "Python bytecode"

The result is obvious.  A million results showed up, most not about this module. 

When the internet started, using a keyword name worked great for SEO.  but now, this module gets lost in the noise.  

A name change would make it possible to learn more about this module. 

In the meantime, lists of links to relevant postings would help. 

But overall, at first glance, this looks like exactly what I need.

**Comment by MatthieuDartiailh** [comment]
I agree the name does not make it easy to find relevant links, but I am not sure how much there is out there beyond the docs. We could add a section about projects using bytecode.
